MeetupPlatformApi/Features/Authentication/RegisterNewUser/RegisterNewUserFeature.cs

```diff
@@ -35,12 +35,21 @@ public async Task<IActionResult> RegisterNewUser([FromBody] RegistrationDto regi
         var user = mapper.Map<User>(registrationDto);
         user.Password = BCrypt.HashPassword(user.Password);
         context.Users.Add(user);
+
+        var refreshTokenId = Guid.NewGuid();
+        var tokenPair = tokenHelper.IssueTokenPair(user, refreshTokenId);
```

Comment:
Consider moving this line of code a little bit lower (closer to the `regitrationResultDto`):
```csharp
var refreshToken = new RefreshToken
{
    Id = Guid.NewGuid(),
    UserId = user.Id    
};
context.RefreshTokens.Add(refreshToken);
await context.SaveChangesAsync();

var tokenPair = tokenHelper.IssueTokenPair(user, refreshToken.Id);
var registrationResultDto = new RegistrationResultDto
{
    UserInfo = mapper.Map<RegistrationResultDto.UserInfoDto>(user),
    TokenPairInfo = mapper.Map<TokenPairDto>(tokenPair)
};
return Created(registrationResultDto);
```